modules/ocf_desktop/files/skel/config/brave/session-store-1

```diff
@@ -0,0 +1,15 @@
+{
+    "settings": {
+      "general.check-default-on-startup": false,
+      "general.is-default-browser": false,
+      "general.homepage": "https://www.ocf.berkeley.edu/about/lab/open-source",
+      "general.startup-mode": "homePage",
+      "shutdown.clear-cache": true,
+      "shutdown.clear-all-site-cookies": true,
+      "shutdown.clear-autofill-data": true,
+      "shutdown.clear-downloads": true,
```

Comment:
Does this delete files in the Downloads directory when it is closed, or does it just clear them from inside the browser itself? The reason I ask is that I don't think we want to remove the files themselves, just the information stored in the browser (so I'm hoping it does that)

Reply:
Just download history. Kinda a misleading variable name. :<

Reply:
Oh ok, that's good then, I was hoping that's what it did.